__tests__/app.test.js

```diff
@@ -160,4 +160,65 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/reviews/:review_id/comments", () => {
+    it("201: POST responds with a 201 code and the comment object that was added. The votes property should also be returned with a value of 0", () => {
+      const requestBody = {
+        username: "dav3rid",
+        body: "Great game! Going to recommend to everyone I know!",
+      };
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(requestBody)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.postedComment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: "Great game! Going to recommend to everyone I know!",
+            votes: 0,
+            author: "dav3rid",
+            review_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+    });
+    it("responds to an invalid review_id with a 400 code and an error message 'Invalid Request", () => {
+      const requestBody = {
+        username: "dav3rid",
+        body: "Great game! Going to recommend to everyone I know!",
+      };
+      return request(app)
+        .post("/api/reviews/not-a-review-id/comments")
+        .send(requestBody)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Invalid Request");
+        });
+    });
+    it("responds to an review_id with no entry in the database with a 404 code and an error message 'Not Found'", () => {
+      const requestBody = {
+        username: "dav3rid",
+        body: "Great game! Going to recommend to everyone I know!",
+      };
+      return request(app)
+        .post("/api/reviews/9999/comments")
+        .send(requestBody)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Not Found");
+        });
+    });
+    it("responds to an invalid request body with a 400 code and an error message 'Invalid Request'", () => {
+      const invalidRequestBody = {
+        invalidUser: "notausername",
+        invalidBody: 10000000,
+      };
```

Comment:
This is 2 tests merged into one. Having a key of Invalid user will trigger your 400 however the username is also not found in the database, which would trigger a 404 if the object was in the valid format. For example: 

const requestBody = {
        username: "mattBishop",
        body: "Great game! Going to recommend to everyone I know!",
      }

Should trigger a 404 because the request is good, it's just the user that is not found in the database 👍